### DISCUSSION TITLE
Start of challenge

### DISCUSSION THREAD

**Discussion by CatInTheRain** [larics/icuas22_competition#51] [Q&A] [answered]
Dear organizers,
I understood that the UAV is only allowed to move after `True` is published on topic `/red/challenge_started`. 
About starting personal nodes which do not involve the movement of the drone, is it allowed to start them before the start signal?

**Comment by fpetric** [comment, marked as answer]
@CatInTheRain thanks for asking, 

you can start your nodes before the message is published on /red/challenge_started. I strongly suggest using waitForSimulation before launching (or atleast waitForRos). As you said, the UAV must not move until the message is published.

**Reply by CatInTheRain** [reply]
Thanks a lot! Thanks also for the hint!
